imagery/i.cluster/testsuite/test_i_cluter.py

```diff
@@ -0,0 +1,294 @@
+import os
+import re
+import math
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+from pathlib import Path
+from grass.script import find_file
+from itertools import combinations
+
+
+class TestIClusterWithSyntheticData(TestCase):
+    group_name = "test_cluster_group"
+    subgroup_name = "test_cluster_subgroup"
+    input_maps = ["synth_map1", "synth_map2", "synth_map3"]
+    signature_file = "sig"
+    num_classes = 3
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up test environment"""
+        cls.use_temp_region()
+        cls.runModule("g.region", n=50, s=0, e=50, w=0, rows=100, cols=100)
+        cls.runModule(
+            "r.mapcalc",
+            expression=f"{cls.input_maps[0]} = if(row() < 25, 10, 5)",
+            overwrite=True,
+        )
+        cls.runModule(
+            "r.mapcalc",
+            expression=f"{cls.input_maps[1]} = if(col() < 25, 20, 10)",
+            overwrite=True,
+        )
+        cls.runModule(
+            "r.mapcalc",
+            expression=f"{cls.input_maps[2]} = row() + col()",
+            overwrite=True,
+        )
+        cls.runModule(
+            "i.group",
+            group=cls.group_name,
+            subgroup=cls.subgroup_name,
+            input=",".join(cls.input_maps),
+        )
+
+    @classmethod
+    def tearDownClass(cls):
+        """Cleanup test environment"""
+        cls.runModule("g.remove", flags="f", type="group", name=cls.group_name)
+        cls.runModule(
+            "g.remove", flags="f", type="raster", name=",".join(cls.input_maps)
```

Comment:
This should work too:
```suggestion
            "g.remove", flags="f", type="raster", name=cls.input_maps
```

Reply:
I have updated the code.